edb/server/compiler/rpc.pyi

```diff
@@ -42,7 +45,10 @@ class SQLParamsSource:
         ...
 
 class CompilationRequest:
-    source: edgeql.Source
+    # FIXME: I didn't want to track down the errors when I put the right
+    # annotation in.
+    # source: edgeql.Source | graphql.Source | pgparser.Source
+    source: typing.Any
```

Comment:
I've fixed this in a commit I've just pushed. I've also missed `SQLParamsSource`